src/Behavior/MillisecondTimestamp.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ipl\Orm\Behavior;
+
+use DateTime;
+use DateTimeZone;
+use ipl\Orm\Contract\PropertyBehavior;
+
+class MillisecondTimestamp extends PropertyBehavior
+{
+    public function fromDb($value, $key, $context)
+    {
+        if ($value !== null) {
+            if (is_string($value)) {
+                $value = (float) $value;
+            }
+
+            $datetime = DateTime::createFromFormaT('U.u', number_format($value / 1000.0, 3, '.', ''));
```

Comment:
There's a capital `T` which should be `t`.

Also, why not just `DateTime::createFromFormat('U.u', sprintf('%F', $value / 1000.0))`?